docs/experiments.rst

```diff
@@ -327,8 +327,9 @@ Currently, simexpal supports the following three ``sbatch`` parameters by using
 the ``experiments.yml``:
 
 - ``procs_per_node``: number of tasks to invoke on each node (slurm: ``--ntasks-per-node=n``)
-- ``num_threads``: number of cpus required per task (slurm: ``-c``, ``--cpus-per-task=ncpus``)
-- ``num_nodes``: number of nodes on which to run (N = min[-max]) (slurm: ``-N``, ``--nodes=N``)
+- ``num_threads``:    number of cpus required per task (slurm: ``-c``, ``--cpus-per-task=ncpus``)
+- ``num_nodes``:      number of nodes on which to run (N = min[-max]) (slurm: ``-N``, ``--nodes=N``)
+- ``exclusive``:      boolean to set the ``--exclusive`` flag, so the job allocation reserves all resources of a node
```

Comment:
If would recommend a phrasing such as: Boolean flag to run an experiment exclusively on specified computing resources. For Slurm, this means we set the ``--exclusive`` flag.